code/game/gamemodes/cult/runes.dm

```diff
@@ -478,15 +478,12 @@ structure_check() searches for nearby cultist structures required for the invoca
 	send_to_playing_players('sound/effects/dimensional_rend.ogg')
 	var/turf/T = get_turf(src)
 	sleep(40)
+	SSticker.mode.eldergod = 0
```

Comment:
No but I came close to changing that and its primarily used for cult_fail += eldergod. 

Since I planned to (but didn't) change the ending flavor text based on whether Nar'Sie "ascended", this felt like the most intuitive way to let the ticker know Nar'Sie's status.